packages/suite/src/hooks/suite/useDeviceModel.ts

```diff
@@ -0,0 +1,6 @@
+import { useSelector } from './useSelector';
+import { TrezorDevice } from '@suite-types/index';
+import { selectDeviceModel } from '@suite-reducers/suiteReducer';
+
+export const useDeviceModel = (overrideDevice?: TrezorDevice) =>
```

Comment:
I guess it's because of the optional param `overrideDevice`, but It's just a syntactic sugar to be able to write just `useDeviceModel(device)` instead of `useSelector(state => selectDeviceModel(state, device));`... But I see no harm in this hook.

Reply:
What stops us from creating a hook for every selector with a param then? :D I think this use of hooks defies the whole idea of hooks, since there is no additional state or effect or ref logic, it it literally an abstraction of `useSelector(state => selectDeviceModel(state, device))`, which  again, implies we can do it in any other situation where we have a param in a selector :( 

When I created the `useBitcoinAmountUnit` hook Dan already mentioned that it can simply be a selector and he was right. My only defence was that the actions were included for convenience but it also meant that if a component only needed to dispatch and action it was still subscribed to the state, so it still is arguable.